lib/rules/no-extra-parens.js

```diff
@@ -428,6 +429,38 @@ module.exports = {
             }
         }
 
+        /**
+         * Check the parentheses around the super class of the given class definition.
+         * @param {ASTNode} node The node of class declarations to check.
+         * @returns {void}
+         */
+        function dryClass(node) {
```

Comment:
Yes, I just used the names as similar to other functions.
I agree that `check` would be better.